### PR TITLE
Ihracp 5614

### PR DESCRIPTION
**Pull request by KMichaelMcGaw**
## JIRA Ticket
- [Title](Url)

#### Major Changes:
- Transcriptions added to `Episode` objects. Returns `None` if there aren't any. 
- The `podcast_transcript` object will be a *list* of dictionaries.

## How To Test
- Include detailed steps on how to test your podcast
- How to replicate issue you are fixing
- How to test the fix
 
## Test Data

- Include relevant data to test, this can be as simple as show ids and as complex as sql queries to set test data.
- Takedown show id: 12345678
- Upsert Show Id: 12345678

## Related PRs

- List related PRs if applicable.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
 # Get the long description from the README file
-with open(path.join(here, 'README.rst'), encoding='utf-8') as f:
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
 setup(
     name='pypodcastparser-ihr',
 
-    version='1.5.0',
+    version='1.6.0.2-dev',
```
Comment (IanNoyes) [inline setup.py]: Can we change this to 1.6.0.2?

Comment (daleyp24) [reply setup.py]: poonam is merging 1.7.0. wait to update

Comment (KMichaelMcGaw) [reply setup.py]: RIP versioning

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (daleyp24) [inline .vscode/settings.json:1]: i dont think we need this file.

Comment (KMichaelMcGaw) [reply .vscode/settings.json:1]: Do you mean the whole `settings.json`? It was added so I can walk through the `testing` feature of VSCode.

File: tox.ini [hunk 3]
```diff
@@ -16,13 +16,14 @@ deps =
...
     pytest-asyncio
+    pytz
 commands =
     python -m coverage run -m pytest --strict {posargs: tests}
     python -m coverage report -m --include="pypodcastparser/*"
 setenv = PYTHONPATH = {toxinidir}/pypodcastparser
 
 [flake8]
-ignore = E501
+ignore = E501, E271, E221, E231, E303, E226, E275, E711,W605 , E225, E203, E128,E124 ,E302,E115,E265,W293,F401,W293,W291,E301
```
Comment (daleyp24) [inline tox.ini]: why are we ignoring so many now?

File: setup.py (hunk 1 above)
Comment (daleyp24) [reply setup.py]: still needs a update

File: tests/test_pyPodcastParser.py [hunk 4]
```diff
@@ -545,6 +545,31 @@ def test_episode_meta_data_episode_season(self):
     def test_episode_parsing_explicit(self):
         self.assertEqual(self.podcast.items[0].itunes_explicit, False)
 
+class TestItunesEpisodeParsingWithTranscription(unittest.TestCase):
```
Comment (daleyp24) [inline tests/test_pyPodcastParser.py:553]: can we add anymore cases?

Comment (KMichaelMcGaw) [reply tests/test_pyPodcastParser.py:553]: What isn't covered?

**Review by daleyp24 — APPROVED** [review]

**Review by DrewJoseyGit — APPROVED** [review]

**Pull request merged**